docs/resource-policy/policy/common-functionality.md

```diff
@@ -131,21 +131,154 @@ to actually mean
 This is not possible with the NRI Reference Plugins configuration CR. Here you
 must use the latter full syntax.
 
-### Cache Allocation Prerequisites
+## Cache Occupancy Monitoring Metrics
 
-Note that for cache allocation control to work, you must have
-- a hardware platform which supports cache allocation
-- resctrlfs pseudofilesystem enabled in your kernel, and loaded if it is a module
-- the resctrlfs filesystem mounted (possibly with extra options for your platform)
+Plugins can be configured to export cache usage prometheus metrics data collected
+by goresctrl. To enable these metrics, make sure that in addition to having RDT
+classes/CLOSes set up in your configuration, you also have
```

Comment:
Is there actually a need to specify any classes in the RDT configuration? Can't it be run in a "discovery" mode to use whatever was configured outside the plugin?

```suggestion
Plugins can be configured to export cache usage as Prometheus metrics. The following configuration options must be specified:
```

Reply:
> Is there actually a need to specify any classes in the RDT configuration? Can't it be run in a "discovery" mode to use whatever was configured outside the plugin?

@marquiz Currently if any class assignments are to be done by the plugin then classes need to be specified (otherwise we'll end up with no resctrl CLOSes/classes, so creating any container assigned to a class will fail in the runtime). But, PTAL #551 which enables running in goresctrl in discovery mode.